ph5/utilities/tests/test_ph5validate.py

```diff
@@ -0,0 +1,113 @@
+'''
+Tests for metadatatoph5
+'''
+import unittest
+from ph5.core import ph5api
+from ph5.utilities import ph5validate
+import os
+
+
+class TestPh5Validate(unittest.TestCase):
+
```

Comment:
If the creation of the PH5file is long and slow you can do it once for all the tests in
```
@classmethod
def setUpClass(cls):
    ...
    self.tmpdir = tempfile.mkdtemp()
    (build the ph5file)
```
and similarly, remove the tmpdir with `tearDownCls`

If it's less than a couple of seconds you can place it in `setUp()` above where the ph5 is opened

You may want to leave the dir if the test fails so it can be inspected.